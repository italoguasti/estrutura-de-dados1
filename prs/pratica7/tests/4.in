100
Bernardo 3 2 1990
Bernardo 2 8 1947
Miguel 19 9 2010
Miguel 28 3 1989
Pedro 8 6 1950
Arthur 5 5 2021
Arthur 26 12 1947
Miguel 5 4 2020
Davi 16 9 1985
Davi 5 5 1951
Sophia 22 4 1954
Bernardo 9 10 2007
Sophia 3 5 1936
Arthur 4 7 1930
Miguel 22 12 1984
Helena 4 4 1970
Arthur 17 9 2018
Pedro 15 8 1961
Gabriel 16 3 1937
Miguel 22 3 1942
Davi 16 4 2010
Sophia 2 12 2020
Isabella 30 3 1941
Arthur 19 2 1992
Sophia 4 8 2008
Pedro 1 10 1935
Bernardo 26 4 1950
Alice 4 7 1955
Davi 22 11 2000
Miguel 8 8 2012
Sophia 14 4 1971
Isabella 11 3 1991
Pedro 16 10 2004
Isabella 25 6 1954
Miguel 9 12 1996
Davi 13 6 1953
Miguel 15 7 1954
Miguel 16 7 1958
Helena 8 7 1985
Isabella 21 6 1963
Gabriel 5 7 2006
Miguel 10 11 1981
Davi 22 2 1985
Sophia 23 12 1931
Pedro 4 4 1938
Sophia 18 5 1976
Alice 12 6 2001
Miguel 22 10 1930
Arthur 29 12 1943
Davi 5 6 1997
Bernardo 5 7 2013
Gabriel 20 7 2016
Arthur 30 1 2001
Davi 6 8 1965
Alice 29 1 1982
Miguel 28 6 1947
Arthur 12 11 1967
Alice 11 9 1983
Gabriel 10 5 1955
Bernardo 6 10 2015
Sophia 4 8 1942
Helena 30 7 1981
Gabriel 15 11 2022
Alice 14 1 1977
Gabriel 9 10 2010
Davi 20 5 1955
Helena 28 8 1982
Davi 18 6 1994
Alice 11 12 1969
Davi 1 2 1949
Isabella 5 5 2021
Miguel 8 3 1976
Miguel 13 3 1949
Isabella 30 6 1932
Miguel 1 9 1967
Helena 15 4 2008
Alice 3 8 1950
Alice 16 11 2019
Davi 22 9 2004
Arthur 25 9 2000
Helena 19 5 1972
Davi 1 2 1967
Pedro 8 6 1978
Bernardo 17 6 1999
Helena 3 8 1945
Bernardo 9 6 1984
Gabriel 13 3 1950
Bernardo 2 5 1956
Sophia 18 2 1955
Arthur 27 8 1986
Alice 27 5 1977
Isabella 11 1 1991
Alice 6 4 2002
Sophia 15 4 1997
Sophia 18 9 2008
Alice 22 12 1993
Gabriel 24 6 1958
Arthur 27 4 1933
Gabriel 16 6 1998
Helena 11 6 1969
