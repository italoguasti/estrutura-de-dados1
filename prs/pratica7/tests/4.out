Analise.....: Comp | Movi
- Bubble....: 4950 | 2404
- Selection.: 4950 | 0096
- Insertion.: 2504 | 2504
